scss/jetpack-deactivate-dialog.scss

```
@@ -0,0 +1,174 @@
+#TB_window.jetpack-disconnect-modal {
+
+	max-width: 635px;
+	max-height: 600px;
+
+	@media screen and (min-width: 661px) {
+		max-height: 360px;
+	}
+
+	#TB_title {
+		font-size: 28px;
+		line-height: 24px;
+		font-weight: 600;
+		height: 24px;
+		padding: 24px;
+		color: #23282d;
+	}
+
+	#TB_ajaxWindowTitle {
+		padding-left: 0px;
+		margin-top: -2px; // compensate line height and vertical align the title
+	}
+
+	#TB_ajaxContent {
+
+		width: auto !important;
+		padding: 0px;
+		max-height: 435px;
+
+		@media screen and (min-width: 661px) {
+			max-height: 203px;
+		}
+
+		.jetpack_deactivation_dialog_content {
+			padding: 24px;
+			padding-bottom: 80px;
+			margin-top: 0px;
+
+			@media screen and (min-width: 661px) {
+				padding-bottom: 24px;
+			}
+
+			p {
+				font-size: 16px;
+				line-height: 24px;
+				margin-top: 0px;
+			}
+
+			ul {
+				font-size: 16px;
+				line-height: 24px;
+				li {
+
+					margin-bottom: 10px;
+
+					span {
+
+						vertical-align: middle;
+
+						&.dashicons {
+							color: red;
+
+						}
+
+					}
+				}
+
+			}
+
+
+		}
+	}
+
+	#TB_closeWindowButton {
+		left: auto;
+		right: 22px;
+		color: #a2a2a2;
```

```suggestion
		color: $wpui-dark-medium-gray;
```

Maybe? That's not the exact same color. I am not sure `#a2a2a2` in on the color palette of wp-admin?